src/main/kotlin/net/atos/zac/app/zaak/model/RestResultaattype.kt

```diff
@@ -47,13 +39,13 @@ fun ResultaatType.toRestResultaatType() = RestResultaattype(
     besluitVerplicht = this.brondatumArchiefprocedure?.afleidingswijze?.let {
         Afleidingswijze.VERVALDATUM_BESLUIT.toValue() == it.name.uppercase(Locale.getDefault()) ||
             Afleidingswijze.INGANGSDATUM_BESLUIT.toValue() == it.name.uppercase(Locale.getDefault())
-    } ?: false,
+    } == true,
```

Comment:
Is this really the same for `null` values?

Reply:
yeah, pretty sure. IntelliJ itself suggested this change as a code improvement. I will check later if this is covered by unit tests. Good to add it any case.